peachjam/js/peachjam.ts

```diff
@@ -76,6 +76,8 @@ class PeachJam {
   setupHtmx () {
     // @ts-ignore
     window.htmx = htmx;
+    // @ts-ignore
+    window.htmx.config.allowNestedOobSwaps = false;
```

Comment:
@longhotsummer @Sandravaphilips  The update modal template uses oob swaps for saving documents in the search view. I've reused it inside the folders list where we return the modals inside the main htmx response. Htmx doesn't like that mixing the two.  This config option disables that behaviour. The other option would have been to create a new update template but this is good for now.